src/main/scala/rocks/muki/graphql/package.scala

```diff
@@ -57,7 +57,9 @@ package object graphql {
     token(Space.? ~> schemaParser)
   }
 
-  private def schemaOrError(label: String, graphQLSchema: GraphQLSchemas): GraphQLSchema =
-    graphQLSchema.schemaByLabel.getOrElse(label, sys.error(s"The schema '$label' is not defined in graphqlSchemas"))
-
+  private def schemaOrError(label: String, graphQLSchema: GraphQLSchemas): Parser[GraphQLSchema] =
+    graphQLSchema.schemaByLabel
+      .get(label)
+      .map(success(_))
+      .getOrElse(failure(s"The schema '$label' is not defined in graphqlSchemas"))
```

Comment:
This fixes the auto-completion behavior with sbt 1.3.x.

Reply:
Nice!